projectmanagement/src/main/java/life/qbic/projectmanagement/application/SortOrder.java

```diff
@@ -0,0 +1,25 @@
+package life.qbic.projectmanagement.application;
+
+/**
+ * A sort order to be consumed by a data providing service.
+ *
+ * @since 1.0.0
+ */
+public record SortOrder(String propertyName, boolean isDescending) {
+
+  public boolean isAscending() {
+    return !isDescending();
+  }
+
+  public SortOrder ascending() {
+    return new SortOrder(propertyName, false);
+  }
+
+  public SortOrder descending() {
+    return new SortOrder(propertyName, true);
+  }
+
+  public static SortOrder of(String propertyName) {
+    return new SortOrder(propertyName, true);
+  }
```

Comment:
This will make the sortorders default value to descending if i understood it correctly. Would it make more sense to also requiring specifying the sort order when creating a new instance?

Reply:
You can specify both over the constructor. For simplicity I provide this method here so that you do not need to decide upon creation on the direction of the sorting. The `descending()` and `ascending()` methods allow you to modify that later on. If you already know both, you can use the constructor `SortOrder(String propertyName, boolean isDescending)`